src/optargpm.h

```diff
@@ -219,6 +219,7 @@ struct OPTARGS {
     FILE *input;
     int   input_index;
     int   record_index;
+    int   free_format;       /* plus-style specs replaced by free format */
```

Comment:
Yes - it's really mostly a flag, but doubles as index. But it is actually superfluous as struct member, so rather than trying to explain that in a comment, I think the easy option is just to eliminate it...

Reply:
Will handle this in a separate PR.